xfree/xc/doc/man/general/XProjectTeam.man
.\" $Xorg: XProjectTeam.cpp,v 1.5 2000/08/21 17:05:21 coskrey Exp $
.\" Copyright (c) 1993, 1994, 1996  X Consortium
.\" Copyright (c) 1996, 2000 The Open Group
.\" 
.\" Permission is hereby granted, free of charge, to any person obtaining a
.\" copy of this software and associated documentation files (the "Software"), 
.\" to deal in the Software without restriction, including without limitation 
.\" the rights to use, copy, modify, merge, publish, distribute, sublicense, 
.\" and/or sell copies of the Software, and to permit persons to whom the 
.\" Software furnished to do so, subject to the following conditions:
.\" 
.\" The above copyright notice and this permission notice shall be included in
.\" all copies or substantial portions of the Software.
.\" 
.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL 
.\" THE X CONSORTIUM BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, 
.\" WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF 
.\" OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE 
.\" SOFTWARE.
.\" 
.\" Except as contained in this notice, the name of the X Consortium shall not 
.\" be used in advertising or otherwise to promote the sale, use or other 
.\" dealing in this Software without prior written authorization from the 
.\" X Consortium.
.\"
.\" $XFree86: xc/doc/man/general/XProjectTeam.man,v 1.2 2001/01/27 18:20:38 dawes Exp $
.\"
.TH X.Org __miscmansuffix__ __xorgversion__
.SH NAME
X.Org, XProjectTeam \- X.Org Group information
.SH SYNOPSIS
Release 6.5 of X Version 11 was brought to you by The X.Org Group.
Release 6.4 of X Version 11 was brought to you by The X Project Team.
.SH DESCRIPTION
The Open Group's X Project Team was created as the successor 
to the X Consortium, Inc., after the X Consortium was merged into
The Open Group. 
The X.Org Group (hereinafter called "X.Org") was created as the successor 
to The X Project Team after the The Open Group ceased operating
The X Project Team.
The purpose of X.Org is to foster development, evolution, and 
maintenance of the X Window System.  X.Org operates under the corporate
umbrella of The Open Group.
.PP
The X Consortium was an independent, not-for-profit Delaware membership
corporation.  It was formed in 1993 as the successor to the MIT X 
Consortium.
.PP
The X Window System was created in the mid-1980s at the Massachusetts
Institute of Technology.  In 1988, MIT formed a member-funded consortium 
to provide the technical and administrative leadership necessary to 
support further development of the X Window System.  In 1992, MIT and 
the membership decided it was in their best interests to move the 
consortium out of MIT and create an independent, stand-alone organization.  
All rights to the X Window System were assigned by MIT to X Consortium, 
Inc. on January 1, 1994.  X Consortium, Inc. closed its doors on December 
31, 1996.  All rights to the X Window System have been assigned to The 
Open Group, for the benefit of the members of X.Org.
.PP
.SH "ADDRESS"
To reach The Open Group public World Wide Web server, use
http://www.opengroup.org/.
.PP
To reach The X.Org public World Wide Web server, use
http://www.x.org/.
.PP
To reach The X.Org public ftp machine, use anonymous ftp at
ftp://ftp.x.org/

.SH "FULL MEMBERS"

.nf
Attachmate
Barco Group
Compaq
Hewlett-Packard
Hummingbird
IBM
Metro Link
Shiman Associates
Silicon Graphics Incorporated
Starnet Communications
Sun Microsystems
The XFree86 Project
US Navy
WRQ
Xi Graphics
.fi


xfree/xc/doc/man/general/XProjectTeam.cpp
.\" $TOG: XProjectTeam.cpp /main/74 1998/02/02 12:52:52 barstow $
.\" Copyright (c) 1993, 1994, 1996  X Consortium
.\" 
.\" Permission is hereby granted, free of charge, to any person obtaining a
.\" copy of this software and associated documentation files (the "Software"), 
.\" to deal in the Software without restriction, including without limitation 
.\" the rights to use, copy, modify, merge, publish, distribute, sublicense, 
.\" and/or sell copies of the Software, and to permit persons to whom the 
.\" Software furnished to do so, subject to the following conditions:
.\" 
.\" The above copyright notice and this permission notice shall be included in
.\" all copies or substantial portions of the Software.
.\" 
.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL 
.\" THE X CONSORTIUM BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, 
.\" WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF 
.\" OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE 
.\" SOFTWARE.
.\" 
.\" Except as contained in this notice, the name of the X Consortium shall not 
.\" be used in advertising or otherwise to promote the sale, use or other 
.\" dealing in this Software without prior written authorization from the 
.\" X Consortium.
.TH XProjectTeam __miscmansuffix__ "Release 6.4"  "X Version 11"
.SH NAME
XProjectTeam \- X Project Team information
.SH SYNOPSIS
Release 6.4 of X Version 11 was brought to you by The Open Group X Project Team.
.SH DESCRIPTION
The Open Group X Project Team was created as the successor to the X
Consortium, Inc., after the X Consortium ceased operations. The purpose 
of The X Project Team is to foster development, evolution, and 
maintenance of the X Window System.
.PP
The X Consortium was an independent, not-for-profit Delaware membership
corporation.  It was formed in 1993 as the successor to the MIT X 
Consortium.
.PP
The X Window System was created in the mid-1980s at the Massachusetts
Institute of Technology.  In 1988, MIT formed a member-funded consortium 
to provide the technical and administrative leadership necessary to 
support further development of the X Window System.  In 1992, MIT and 
the membership decided it was in their best interests to move the 
consortium out of MIT and create an independent, stand-alone organization.  
All rights to the X Window System were assigned by MIT to X Consortium, 
Inc. on January 1, 1994.  X Consortium, Inc. closed its doors on December 
31, 1996.  All rights to the X Window System have been assigned to the 
Open Group.
.PP
.SH X PROJECT TEAM STAFF
.nf

Arthur Barstow
Kaleb Keithley
Sekhar Makkapati
M. S. Ramesh
Ken Flowers
Dave Knorr
.fi

.SH "ADDRESS"
To reach The Open Group public Wide World Web server, use
http://www.opengroup.org/ and http://www.opengroup.org/desktop/x/.
.PP
To reach The X Project Team public ftp machine, use anonymous ftp at
ftp://ftp.x.org/

.SH "FULL MEMBERS"

.nf
Astec
Attachmate
BARCO Chromatics
CliniComp International
CRM
Digital
Fujitsu
Hewlett-Packard
Hitachi
Hummingbird Communications
IBM
Insignia
Jupiter Systems
Metro Link
NCD
NetManage
Peritek
Seaweed Systems
Sequent Computer Systems
Shiman Associates
Silicon Graphics
Siemens Nixdorf
Societe Axel
SunSoft
Vigra - Visicom Laboratories
WRQ
Xi Graphics
.fi
